src/Loggers/OwnFilteringChannel.cpp

```diff
@@ -0,0 +1,51 @@
+#include "OwnFilteringChannel.h"
```

Comment:
This seems like not finished yet.

Reply:
I just removed the commented-out debug statements. Was that what you were referring to?

Reply:
Ah yes, I will have another look then today.